Sources/ApodiniDatabase/Handlers/Create.swift

```diff
@@ -0,0 +1,30 @@
+import Fluent
+import Apodini
+
+/// A Handler that creates, if possible, an object in the database that conforms to `DatabaseModel` out of the body of the request.
+/// It uses the database that has been specified in the `DatabaseConfiguration`.
+public struct Create<T: DatabaseModel>: Handler {
+    @Apodini.Environment(\.database)
+    private var database: Fluent.Database
+    
+    @Parameter
+    private var object: T
+
+    public func handle() -> T? {
+        // swiftlint:disable all
```

Comment:
Please don't disable all SwiftLint warnings. Please only disable a single rule and provide a detailed description on why the rule is disabled in a comment [as described in this message](https://github.com/Apodini/Apodini/pull/83#issuecomment-752128774)

In general I think it would be good to support throwing handlers that would be able to forward throwing errors to the client. Vapor supports this mapping of errors to networking errors and correct responses to the client. I think this would be a great addition to Apodini and the database code would be a prime example of the usage of throwing errors.